packages/app-layout/src/vaadin-drawer-toggle.js

```diff
@@ -56,10 +56,11 @@ class DrawerToggle extends Button {
           top: 12px;
         }
       </style>
-      <slot>
+      <slot name="icon">
         <div part="icon"></div>
       </slot>
       <slot name="tooltip"></slot>
+      <slot></slot>
```

Comment:
Apps that have correctly configured a custom icon for the drawer toggle would see 2 icons (Requires a change on the Flow component side also):

<img width="400" alt="Screenshot 2022-09-21 at 9 41 24" src="https://user-images.githubusercontent.com/1222264/191432766-763bb930-5722-4ff6-959f-67f43258a1bc.png">

To avoid a breaking change, we could instead add a slot change listener to the default slot. In case the content is just white-space, it could fall-back to a default icon for example...

```html
<slot>
  <div part="icon"></div>
</slot>
<div part="icon" hidden="[[!__showFallbackIcon]]"></div>

<slot name="tooltip"></slot>
```

Reply:
Or if the default slot just contains whitespace, the component could call `this.innerHTML = ''`

Reply:
Or if the default slot contains only whitespace, the component could call `this.innerHTML = ''`

Reply:
Changed the PR to use the fallback icon as you suggested and `FlattenedNodesObserver` to toggle it.